services/workflows-service/docker-compose.yml

```diff
@@ -15,7 +15,8 @@ services:
       DB_USER: ${DB_USER}
       DB_PASSWORD: ${DB_PASSWORD}
       DB_PORT: ${DB_PORT}
-      SESSION_SECRET: ${SESSION_SECRET}iGdnj4A0YOhj8dHJK7IWSvQKEZsG7P70FFehuddhFPjtg/bSkzFejYILk4Xue6Ilx9y3IAwzR8pV1gb4
+      SESSION_SECRET: ${SESSION_SECRET}
```

Comment:
Won't it break existing environments?

Reply:
I know, but still, it will break existing environments, right?

Reply:
not sure we are using docker compose to run our service on our customer